community/lexicon/bookmarks/bookmark.json

```diff
@@ -0,0 +1,35 @@
+{
+  "lexicon": 1,
+  "id": "community.lexicon.bookmarks.bookmark",
+  "defs": {
+    "main": {
+      "type": "record",
+      "description": "Record bookmarking a link to come back to later.",
+      "key": "tid",
+      "record": {
+        "type": "object",
+        "required": [
+          "subject",
+          "createdAt"
+        ],
+        "properties": {
+          "subject": {
+            "type": "string",
+            "format": "uri"
+          },
+          "createdAt": {
+            "type": "string",
+            "format": "datetime"
+          },
+          "tags": {
+            "type": "array",
+            "description": "Tags for content the bookmark may be related to, for example 'news' or 'funny videos'",
+            "items": {
+              "type": "string"
+            }
+          }
```

Comment:
```suggestion
```

✂️

Reply:
why remove tags?

Reply:
Oh, I thought that was where the discussion was going. Def OK with leaving them!